public/js/common/export/export.js

```diff
@@ -17,8 +17,11 @@ function createExportModalDiv() {
     var img = (context === 'graph') ? getGraphImage() : getGridImage(session);
     var contentDiv = $('<div></div>');
     var topContentDiv = $('<div></div>');
+    var calendarOption = $('<a href="http://localhost:8000/calendar">Download ICS</a>')
     contentDiv.attr('id', 'modal-content-container');
+    calendarOption.attr('target', '_blank');
     topContentDiv.html('<img id="post-image" src="data:image/png;base64,' + img + '" />');
+    contentDiv.append(calendarOption);
     contentDiv.append(topContentDiv);
```

Comment:
Due to jQuery's [fluent interface](https://en.wikipedia.org/wiki/Fluent_interface), you may be able to 'chain' these calls.